{
  try {
    fut.cancel();
  }
 catch (  GridException e) {
    U.error(log,"Failed to cancel task: " + fut,e);
  }
}
